addons/web/static/src/legacy/js/widgets/colorpicker.js

```diff
@@ -113,6 +113,7 @@ var ColorpickerWidget = Widget.extend({
         if (rgba) {
             this._updateRgba(rgba.red, rgba.green, rgba.blue, rgba.opacity);
         }
+        setTimeout(() => this._updateUI());
```

Comment:
Why setTimeout? And should it be fix commit?

Reply:
Extracted to a distinct commit.
The components width and height used to position the color picker pointer and the color and opacity sliders are 0 until the components have actually been rendered, therefore we rely on the `setTimeout` so that the width and height have their actual layout-based values.